packages/design-picker/src/available-designs-config.json

```diff
@@ -424,6 +424,17 @@
 			"is_premium": false,
 			"is_fse": true,
 			"features": []
+		},
+		{
+			"title": "Start with an empty page",
+			"slug": "blank-canvas",
+			"template": "blank-canvas",
+			"theme": "blank-canvas",
+			"categories": [ "featured" ],
+			"is_premium": false,
+			"is_fse": false,
+			"is_alpha": true,
+			"features": []
```

Comment:
This is here to allow for testing, but it could be kept if we want to fix #51979 with this PR too